server/scripts/seed.ts

```diff
@@ -0,0 +1,36 @@
+import Db from 'server/lib/db';
+import log from 'server/lib/log';
+import User from 'server/models/User.model';
+import { makeUser } from 'server/test/utils';
+import { UserObject } from 'server/types/User';
+
+async function create100Users(): Promise<UserObject[]> {
+  const promises = Array.from({ length: 100 }, makeUser);
+  const userArray = await Promise.all(promises);
+  return userArray;
+}
+
+async function insertUsers(userArray: UserObject[]) {
+  try {
+    await Db.sequelize.transaction(async transaction => {
+      await User.bulkCreate(userArray, { transaction });
+    });
+    log('Successfully inserted 100 users');
+  } catch (error) {
+    console.log(error);
+    log('Error inserting users');
+  }
+}
+
+export async function seed(): Promise<void> {
+  try {
+    const userArray = await create100Users();
+    await insertUsers(userArray);
```

Comment:
any reason why these are split up?  is there any reason we would want to generate multiple user creation objects, but _not_ insert them?  if not, then i think it would be appropriate to just do the insertion in the same function that "creates" the users

Reply:
It's essentially a 1-liner after a quick refactor, I'd throw it in `Utils`, honestly. Might be handy if we do anything with accessors or mutators where we want to have the raw data for the users prior to insertion.

Reply:
I'm not sure what you are referencing here?

Reply:
ok yeah i can combine them